cmake/merge_archives_unix.cmake

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2009 Sun Microsystems, Inc.
+# Copyright (c) 2020 IBM
```

Comment:
Would be pity to remove the file after than ;)?

Reply:
Having been burnt in reviews for not going far enough I'd love to remove more :-)